sources/packages/backend/apps/db-migrations/src/sql/Queue/Add-cas-send-invoices.sql

```diff
@@ -0,0 +1,19 @@
+INSERT INTO
+  sims.queue_configurations(queue_name, queue_configuration, queue_settings)
+VALUES
+  (
+    'cas-send-invoices',
+    '{
+        "cron": "0 5,12,17 * * 1-5",
+        "retry": 3,
+        "cleanUpPeriod": 2592000000,
+        "retryInterval": 180000,
+        "dashboardReadonly": false,
+        "pollingRecordLimit": 2000
```

Comment:
Where is this limit being applied to ensure no more than this amount will be sent to CAS? Can you please review the implementation of the notifications and follow the same approach?